Branch

alekslevin opened the pull request:
made modifications because jonathan wanted me to

Commit:
added lines

added lines to help navigate app structure

Commit:
changed branch name

changed branch name

Commit:
removed unnecessary files

removed unnecessary files

jtschoonhoven commented on capture_the_flag/app.py:
```diff
@@ -1,7 +1,10 @@
-from flask_restful import Api
+import os
+
+os.chdir('<parent-directory>')
```
This breaks the app.

jtschoonhoven reviewed (changes requested):
The imports are organized according to PEP8. Please don't change them without cause.
https://www.python.org/dev/peps/pep-0008/#imports

Commit:
Delete workspace.xml

Commit:
Create README.md

Commit:
Create README.md

Commit:
modified spacing and removed module import

modified spacing and removed database module import from
capture_the_flag.models.user

Commit:
Merge remote-tracking branch 'origin/master' into branch

ghost commented:
made requested changes

jtschoonhoven commented on capture_the_flag/app.py:
```diff
@@ -1,11 +1,10 @@
 from capture_the_flag.models.user import load_user_from_session, User
 from capture_the_flag.session import SuperSecureSessionInterface
 from capture_the_flag.routes import files, index, login, logout, signup, users
+from flask_restful import Api
```
please put this back with the other 3rd party imports

jtschoonhoven commented on capture_the_flag/database.py:
```diff
@@ -1,5 +1,4 @@
 import sqlite3  # noqa
-
```
>You should put a blank line between each group of imports.

please don't delete newlines between groups of imports

jtschoonhoven commented on capture_the_flag/models/base.py:
```diff
@@ -1,7 +1,5 @@
 from datetime import datetime
-
```
> You should put a blank line between each group of imports.

please don't delete newlines between groups of imports

jtschoonhoven commented on capture_the_flag/models/user.py:
```diff
@@ -1,10 +1,7 @@
 import random
 import time
-
```
> You should put a blank line between each group of imports.

please don't delete newlines between groups of imports

jtschoonhoven commented on capture_the_flag/routes/files.py:
```diff
@@ -1,5 +1,4 @@
 import os
-
```
> You should put a blank line between each group of imports.

please don't delete newlines between groups of imports

jtschoonhoven commented on requirements.txt:
```diff
@@ -1,4 +1,6 @@
+#Python 2.7
```
This should go in `setup.py`, not requirements.txt. The requirements themselves are not specific to any Python version.

jtschoonhoven reviewed (changes requested):
https://www.python.org/dev/peps/pep-0008/#imports

> Imports should be grouped in the following order:
> 
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports
> 
> You should put a blank line between each group of imports.

jtschoonhoven commented:
Also, please don't check in `database.db`

Commit:
changes for pep compliance

changes for pep compliance

ghost commented:
yes. definitely

Commit:
changes for pep compliance

changes for pep compliance
